### DISCUSSION TITLE
Which starter scripts to include?

### DISCUSSION THREAD

**Discussion by johnlindquist** [johnlindquist/kit#45] [Q&A]
<img width="635" alt="Screen Shot 2021-02-24 at 12 09 56 PM" src="https://user-images.githubusercontent.com/36073/109052838-513da380-7699-11eb-8eaa-02361992a1c5.png">

Is this enough? Should we include some more fancy scripts to get people started?

I'd like to encourage people to visit simplescripts.dev install/share scripts, but also want them to have a nice experience as they're just getting started.

**Comment by cfritze64** [comment]
I think I'd include a script with an API call. How http requests are implemented varies greatly between different Javascript environments, and there's often confusion about whether the implementation is async or not, whether one has to test for a return, promises, etc. So straightforward here, so show it off.

Then maybe a script that demonstrates how AppleScript is called and how to access what is returned? One of the Chrome tabs things or a window positioner?

Something that demonstrates file read/write?

And a script that has the header filled out - 
Shortcut:
Menu:
Description:

**Reply by johnlindquist** [reply]
Those ideas make a lot of sense, thanks.
